express/socket.js

```diff
@@ -125,12 +125,17 @@ module.exports = (io) => {
                 return;
             }
             // Make room active
-            let query = { roomNumber: socket.room };
-            Room.findOneAndUpdate(query, { isActive: true }, { new: true }).then(result => {
+            const query = { roomNumber: socket.room, creatorId: socket.id };
+
+            try {
+                const result = await Room.findOneAndUpdate(query, { isActive: true }, { new: true })
```

Comment:
haha yeah we should probably make everything use `await`. This is much cleaner, thanks!
But I think we forgot to make the function asynchronous ;) 
```
               socket.on("initiate swiping", async () => {
```

Reply:
Aha, yes! I'm spoiled by ESLint yelling at me to mark functions as `async` in other projects ^_^